src/Stratiteq.Extensions.AspNetCore/ProblemDetailsWithErrorCode.cs

```diff
@@ -4,6 +4,7 @@
 
 using Microsoft.AspNetCore.Mvc;
 using System;
+using Newtonsoft.Json;
```

Comment:
Can this dependency be avoided by using System.Text.Json instead?

Reply:
The package targets Netstandard 2.0 which does not seem to include it.